### PR TITLE
fix: stay selected left menu when cluster namespace changed

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by erdkse** [comment]
- also fixes Dashboard Pane filter, Couldn't find an issue for it

**Comment by erdkse** [comment]
https://user-images.githubusercontent.com/2957624/230819988-78357663-2c14-49ee-8378-eaa86da5519c.mov

File: src/components/organisms/DashboardPane/DashboardPane.tsx [hunk 1]
```diff
@@ -99,6 +113,46 @@ const DashboardPane = () => {
...
+    [clusterResourceMeta, compareNamespaces]
+  );
+
+  const getResourceCount = useCallback((kind: string) => getResources(kind).length, [getResources]);
+
+  const getProblemCount = useCallback(
+    (kind: string, level: 'error' | 'warning') => {
+      return getResources(kind).reduce((total: number, resource: ResourceMeta) => {
+        const problemCount = problems
+          .filter(p => p.level === level)
```
Comment (topliceanurazvan) [inline src/components/organisms/DashboardPane/DashboardPane.tsx:142]: Can we do just one single filtering in here so we don't iterate twice the same array?

**Review by topliceanurazvan — CHANGES_REQUESTED** [review]

**Comment by topliceanurazvan** [comment]
Also as a nice have we could add a skeleton loading in the list of items while the namespace is being changed

**Review by topliceanurazvan — APPROVED** [review]

**Pull request merged**